src/par/src/autocluster.cpp

```diff
@@ -173,15 +173,20 @@ Metric AutoClusterMgr::computeMetrics(dbModule* module)
   unsigned int num_macro = 0;
 
   for (dbInst* inst : module->getInsts()) {
+    dbMaster* master = inst->getMaster();
     const LibertyCell* liberty_cell = network_->libertyCell(inst);
+    // add for designs with pads
+    // check if the instance is a pad or empty block (such as marker)
```

Comment:
What is a "marker"?  Any cell without a liberty must be skipped, otherwise the next line will segv, not just isBlock()